docs/build-on-polkadot/parachains/asset-hub/register-a-local-asset.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Register a Local Asset
+description: Comprehensive guide to registering a local asset on the Asset Hub system parachain, including step-by-step instructions.
+---
+
+# Register a Local Asset on Asset Hub
+
+## Introduction
+As detailed in the [Asset Hub Overview](./overview.md#local-assets){target=_blank} page, Asset Hub accommodates two types of assets: local and foreign. Local assets are those that were created in Asset Hub and are identifiable by an integer ID. This guide will take you through the steps of registering a local asset on the Asset Hub parachain.
+
+## Prerequisites
+
+Before you begin, ensure you have access to the [Polkadot.js Apps](https://polkadot.js.org/apps/){target=_blank} interface and a funded wallet with DOT or KSM.
+
+- For Polkadot Asset Hub, you would need a `deposit of 10 DOT` and around `0.201 DOT for the metadata`
+- For Kusama Asset Hub, the `deposit is 0.1 KSM` and around `0.000669 KSM for the metadata`
+
+You need to ensure that your Asset Hub account balance is a bit more than the sum of those two deposits, which should seamlessly account for the required deposits and transaction fees.
+
+## Steps to Register a Local Asset
+
+To register a local asset on the Asset Hub parachain, follow these steps:
+
+1. Open the [Polkadot.js Apps](https://polkadot.js.org/apps/){target=_blank} interface and connect to the `Asset Hub` parachain
+      - You may prefer to test local asset registration on TestNet before registering the asset on a MainNet hub. If you still need to set up a local testing environment, review the [Environment setup](#environment-setup) section for instructions. Once the local environment is set up, connect to the `Local Node (Chopsticks)` available on `ws://127.0.0.1:8000`. 
```

Comment:
Why is `Local Node (Chopsticks)` in back ticks, is this a UI element? If so, it should be in bold instead